.idea/runConfigurations/Artemis__Server_.xml

```diff
@@ -0,0 +1,10 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Artemis (Server)" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot">
+    <module name="Artemis.main" />
+    <option name="SPRING_BOOT_MAIN_CLASS" value="de.tum.in.www1.artemis.ArtemisApp" />
+    <option name="VM_PARAMETERS" value="-Dspring.profiles.active=dev,bamboo,bitbucket,jira,artemis" />
```

Comment:
```suggestion
    <option name="ACTIVE_PROFILES" value="dev,bamboo,bitbucket,jira,artemis" />
```

Reply:
I haven't used the `ACTIVE_PROFILES` option on purpose as the option is only available in IntelliJ Ultimate and some of the developers only use the Community Edition. Therefore I will not add your suggestion for this and the other files :-)